bootstrap/lib/cache/index.js

```diff
@@ -47,13 +46,23 @@ const envOptions = () => ({
   },
 })
 
-const withCache = (execute, options) => {
+const getCacheImpl = async (options) => {
+  switch (options.type) {
+    case 'redis':
+      return await redis.RedisCache.build(options.redis)
```

Comment:
Unneeded async usage
```suggestion
const getCacheImpl = (options) => {
  switch (options.type) {
    case 'redis':
      return redis.RedisCache.build(options.redis)
```